woodwork/tests/accessor/test_serialization.py

```diff
@@ -480,17 +504,18 @@ def test_to_disk_custom_data_filename(sample_df, tmpdir, file_format):
             path=str(tmpdir), format=file_format, filename=data_filename
         )
         assert os.path.isfile(os.path.join(tmpdir, "data", filename_to_check))
-        deserialized_df = read_woodwork_table(path=str(tmpdir), filename=data_filename)
+        deserialized_df = read_woodwork_table(
+            path=str(tmpdir),
+            filename=data_filename,
+        )
         pd.testing.assert_frame_equal(
             to_pandas(sample_df, index=sample_df.ww.index, sort_index=True),
             to_pandas(deserialized_df, index=deserialized_df.ww.index, sort_index=True),
         )
         assert sample_df.ww.schema == deserialized_df.ww.schema
 
 
-@pytest.mark.parametrize(
-    "file_format", ["csv", "pickle", "parquet", "arrow", "feather", "orc"]
-)
+@pytest.mark.parametrize("file_format", ["csv", "pickle", "arrow", "feather", "orc"])
```

Comment:
Since we took parquet out of this, do we have other test coverage to ensure that custom typing changes we've made to the dataframe are preserved when we write to and then read from parquet?

Reply:
Ah, the intent of this test is to confirm that if a user specifies something for `typing_info_filename` in the `ww.to_disk` call that we can properly deserialize.

With this PR, parquet no longer writes a separate typing info JSON file so we don't need to test this behavior any more.

Reply:
Got it.